web/frontend/package.json

```diff
@@ -2,52 +2,58 @@
   "name": "@viamrobotics/remote-control",
   "version": "0.1.37",
   "license": "Apache-2.0",
-  "files": [
-    "src/components/**/*.vue",
-    "src/lib/**/*.ts"
-  ],
+  "type": "module",
+  "files": ["dist"],
+  "module": "./dist/rc.js",
+  "types": "./dist/rc.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/rc.js"
+    }
+  },
   "dependencies": {
```

Comment:
Dependencies / DevDependencies have been reorganized according to what's shared with app. This will ultimately result in node_modules for app being smaller due to unnecessary packages not being downloaded.